File: 7-kyu/coding-meetup-#2---higher-order-functions-series---greet-developers/python/solution.py
def greet_developers(lst):
    ret = []
    for dct in lst:
        to_append = dct
        to_append['greeting'] = f'Hi {dct["firstName"]}, what do you like the most about {dct["language"]}?'
        ret.append(to_append)
    return ret

File: 7-kyu/coding-meetup-#2---higher-order-functions-series---greet-developers/javascript/solution.js
const greetDevelopers = (list) => {
  let ret = [];
  for (const obj of list) {
    let temp = obj;
    temp.greeting = `Hi ${obj.firstName}, what do you like the most about ${obj.language}?`;
    ret.push(temp);
  }
  return ret;
};
